core/chains/evm/txmgr/eth_broadcaster_test.go

```diff
@@ -1926,9 +1926,9 @@ func TestEthBroadcaster_IncrementNextNonce(t *testing.T) {
 	keyState, _ := cltest.MustInsertRandomKeyReturningState(t, ethKeyStore, 0)
 
 	// Cannot increment if supplied nonce doesn't match existing
-	require.Error(t, ethKeyStore.IncrementNextSequence(evmtypes.NewAddress(keyState.Address.Address()), &cltest.FixtureChainID, int64(42)))
+	require.Error(t, ethKeyStore.IncrementNextSequence(evmtypes.NewAddress(keyState.Address.Address()), &cltest.FixtureChainID, evmtypes.Nonce(int64(42))))
```

Comment:
nit/ can this conversion be omitted? Plain `42` is an [untyped constant](https://go.dev/ref/spec#Constants) that should be directly convertible to `Nonce` IIUC
```suggestion
	require.Error(t, ethKeyStore.IncrementNextSequence(evmtypes.NewAddress(keyState.Address.Address()), &cltest.FixtureChainID, evmtypes.Nonce(42)))
```